src/App.js

```diff
@@ -26,6 +27,12 @@ const VotingPage = lazy(() => import("./features/Voting/voting"));
 const StatsPage = lazy(() => import("./features/Stats/stats"));
 
 function App() {
+  const [openWrongChainModal, setOpenWrongChainModal] = useState(false);
+
+  const wrongChainCtx = {
+    openWrongChainModal,
+    setOpenWrongChainModal
+  };
```

Comment:
I would suggest to wrap it into a Provider component like the others, otherwise this definitions would make the App component hard to read in the future

Reply:
@kesar let's see if that can be done ;)